### ISSUE TITLE
Authenticator Reset Password Using Phone Number is Missing Country Code Field

### ISSUE THREAD

**Issue by walkinjack** [aws-amplify/amplify-ui#1512]
### Before creating a new issue, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

React

### Which UI component?

Authenticator

### How is your app built?

Next.js

### Please describe your bug.

The country code dropdown menu is missing on the password reset component.

### What's the expected behaviour?

Users who authenticate with a phone number can select their country code on the reset password component.

### Help us reproduce the bug!

1. Add the auth component to an amplify project that only allows signing in using a phone number. 
2. Add the Authenticator component to the project
3. Sign up using a phone number
4. Click the "Forgot Password Link"
5. The phone field is missing the Country Code drop down menu.

### Code Snippet

```javascript
// Put your code below this line.
import Amplify from 'aws-amplify';
import awsconfig from './aws-exports';
import { Authenticator } from '@aws-amplify/ui-react';

Amplify.configure(awsconfig);

export default function App() {
  return (
    <Authenticator signUpAttributes={[]}>
      {({ signOut, user }) => (
        <main>
          <h1>Hello {user.username}</h1>
          <button onClick={signOut}>Sign out</button>
        </main>
      )}
    </Authenticator>
  );
}
```

### Additional information and screenshots

![amplify-reset-password-phone](https://user-images.githubusercontent.com/92527372/157969721-8de35a39-e230-4314-8633-3c46707b5cb3.png)

**Comment by ErikCH** [comment]
Thanks @walkinjack ! You are right. We are looking to add some new features for users that are using email and phone. We'll add this to our backlog.

**Comment by wlee221** [comment]
This has been released to `@aws-amplify/ui-react@2.10.1`. You should be able to use https://www.dev.ui.docs.amplify.aws/components/authenticator#form-field-customization to customize it as well.

Please let us know if this doesn't work!
